### PR TITLE
Allow custom content in Alert.

### PR DESCRIPTION
**Pull request by PavelHolec**
image

### CODE REVIEW

File: Sources/Orbit/Components/Alert.swift [hunk 1]
```diff
@@ -18,15 +30,16 @@ import SwiftUI
...
 /// - Note: [Orbit definition](https://orbit.kiwi/components/alert/)
 /// - Important: Component expands horizontally to infinity.
-public struct Alert: View {
+public struct Alert<Content: View>: View {
 
     let title: String
     let description: String
     let icon: Icon.Symbol
-    let buttons: Buttons
+    let buttons: AlertAction
```
Comment (sjavora) [inline Sources/Orbit/Components/Alert.swift:38]: Shouldn't this be `action: AlertAction` now?

File: Sources/Orbit/Components/Alert.swift [hunk 2]
```diff
@@ -42,13 +55,18 @@ public struct Alert: View {
...
+                
+                switch buttons {
+                    case .primary, .secondary, .primaryAndSecondary:
+                        // Keeping the identity of buttons for correct animations
+                        buttonsView
+                    case .none:
+                        EmptyView()
+                }
             }
+            .padding(.leading, icon == .none ? 0 : 28)
```
Comment (sjavora) [inline Sources/Orbit/Components/Alert.swift:69]: Is there no constant for 28?

**Review by sjavora — CHANGES_REQUESTED** [review]

File: Sources/Orbit/Components/Alert.swift (hunk 2 above)
Comment (PavelHolec) [reply Sources/Orbit/Components/Alert.swift:69]: Sadly no

File: Sources/Orbit/Components/Alert.swift (hunk 1 above)
Comment (PavelHolec) [reply Sources/Orbit/Components/Alert.swift]: Reverted the name to AlertButton.

**Review by sjavora — APPROVED** [review]

**Pull request merged**